custom/icds/tasks/sms.py

```diff
@@ -51,3 +55,27 @@ def send_monthly_sms_report():
         send_html_email_async.delay(subject, recipients, message,
                                     email_from=settings.DEFAULT_FROM_EMAIL)
         raise e
+
+
+@receiver(post_save, sender=DailyOutboundSMSLimitReached)
+def send_sms_limit_exceeded_alert(sender, instance, **kwargs):
+    if settings.SERVER_ENVIRONMENT == 'icds':
+        sms_daily_limit = 10000
+        domain_obj = Domain.get_by_name(instance.domain)
+        if domain_obj:
+            sms_daily_limit = domain_obj.get_daily_outbound_sms_limit()
+
+        subject = "SMS Daily Limit exceeded"
+        recipients = [
+            'ndube@dimagi.com',
+            'dsivaramakrishnan@dimagi.com',
+            'mshashtri@dimagi.com',
+            'asharma@dimagi.com'
+        ]
+        message = _("""
+        Hi,
+        This is to inform you that the Daily SMS limit for domain {domain} has exceeded for {date}.
+        The current limit set is {sms_limit}
```

Comment:
nit: The limit set was {sms_limit}
In case an email is viewed in future.

Reply:
Or we can use
`The sms limit at the time of sending this email is set to {sms_limit}` 
What say?

Reply:
nit: `was set to`